Wikipedia/Code/EventLoggingFunnel.h

```diff
@@ -118,6 +118,8 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (NSString *)wmf_sessionID;
 
+- (NSString *)timestamp;
```

Comment:
if this is `@property (nonatomic, readonly) NSString *timestamp;` it'll allow you to use it as `.timestamp` instead of `.timestamp()` in Swift